packages/integration-tests/suites/public-api/captureMessage/with_level/init.js

```diff
@@ -0,0 +1,11 @@
+import * as Sentry from '@sentry/browser';
+
+window.Sentry = Sentry;
+window.events = [];
+
+Sentry.init({
+  dsn: 'https://public@dsn.ingest.sentry.io/1337',
+  beforeSend: event => {
+    window.events.push(event);
```

Comment:
we can remove this

Reply:
@AbhiPrasad, I have opened a PR for an alternative implementation for getting multiple events into tests without this hack. could you take a look? -> https://github.com/getsentry/sentry-javascript/pull/4355